npm/pkg/dataplane/testfiles/npmgr.json

```diff
@@ -0,0 +1,1579 @@
+{
```

Comment:
instead of "testfiles", call test folders "testdata"
https://pkg.go.dev/cmd/go/internal/test
> The go tool will ignore a directory named "testdata", making it available
to hold ancillary data needed by the tests.